platform/core/src/services/HangingProtocolService/HangingProtocolService.js

```diff
@@ -35,11 +36,21 @@ class HangingProtocolService {
     Object.assign(this, pubSubServiceInterface);
   }
 
+  listenThemeProtocols(point) {
+
+    this.addProtocols(point.protocols);
+  }
+
   reset() {
     this.studies = [];
     this.protocols = [];
     this.hpAlreadyApplied = [];
     this.matchDetails = [];
+
+    if (!this._listenThemeProtocols) {
```

Comment:
Load any protocols specified by the theme properties, which allows setting up custom sets of hanging protocols for specific uses or users - that is, as a user configurable item rather than hard coded in the extensions modules.